src/test/forward_index_test.cpp

```diff
@@ -128,9 +128,25 @@ int forward_index_tests()
     num_failed += testing::run_test("forward-index-read-file-corpus", [&]()
     {
         ceeaus_forward_test();
-        system("rm -rf ceeaus-* test-config.toml");
     });
 
+    num_failed += testing::run_test("forward-index-build-uninvert", [&]()
+    {
+        system("rm -rf ceeaus-*");
+
+        // hack to inject "uninvert = true" at the top of the config file
+        auto cfg_contents = filesystem::file_text("test-config.toml");
+        cfg_contents = "uninvert = true\n" + cfg_contents;
+        filesystem::delete_file("test-config.toml");
```

Comment:
Re unit tests: the class labels are still strings, always. For regression, we'd have a different metadata field that would be a double pulled from the metadata table.

The double values I'm referring to are the actual values stored as the counts data in the postings file, as well as the "total counts" value stored in the postings file (which, for `inverted_index` gives you the total term occurrence count in the entire corpus, and for `forward_index` now gives you the sum of all of the weights for all of the terms in that document, which we can use for e.g. normalization).